buildkite/bazelci.py

```diff
@@ -2339,12 +2339,12 @@ def set_env_var(config_key, env_var_name):
         pipeline_steps.append(step)
 
     if skipped_due_to_bazel_version:
-        lines = ["The following tasks were skipped since they require specific Bazel versions:", ""]
-        lines += ["- {}".format(s) for s in skipped_due_to_bazel_version]
+        lines = ["\n- {}".format(s) for s in skipped_due_to_bazel_version]
         commands = [
             "buildkite-agent annotate --style=info '{}' --append --context 'ctx-skipped_due_to_bazel_version'".format(
-                "\n".join(lines)
-            )
+                "".join(lines)
+            ),
+            "buildkite-agent meta-data set 'has-skipped-steps' 'true'",
```

Comment:
This entire section is so ugly, including the fact that our `create_step` API takes commands to run as simple strings instead of passing argv as list elements. I love it. 😆